ruby/lib/rspec/queue.rb

```diff
@@ -148,6 +149,54 @@ def queue_config
 
     RSpec::Core::Parser.prepend(ParserExtension)
 
+    module ExampleExtension
+      protected
+
+      def mark_as_requeued!(reporter)
+        @exception = nil
+        @metadata = @metadata.dup
+        @metadata[:execution_result] = execution_result.dup
+        execution_result.pending_message = "Failed but was requeued" # TODO: Improve message
```

Comment:
"The example failed, but another attempt will be done to rule out flakiness" ?

Reply:
👍

Reply:
I'd like to see if I can include the failure details as well.